### PR TITLE
Simplify folder structure

### PR DESCRIPTION
**Pull request by li-kai**
Changes are by commits

In summary
- remove /provisioning and /api
- rename /www to /website
- hoist /www/src/js to /website/src
- drop unnecessary .gitattribute settings

### CODE REVIEW

**Comment by ZhangYiJiang** [comment]
Is CI broken by this change?

**Comment by ZhangYiJiang** [comment]
Oh, it looks like the test passed, just that CircleCI didn't report it to GitHub. That's fine, you can override the CI check in that case https://circleci.com/workflow-run/940821d8-0c4d-4450-bbeb-b4282b927431

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (ZhangYiJiang) [inline .vscode/settings.json]: Just to double check, we are deliberately checking this in?

File: export/README.md [hunk 2]
```diff
@@ -75,30 +75,30 @@ Returns the HTML content of the page that Puppeteer renders.
...
 # Deploy the files to production - optionally add --dry-run to check which files are changed first
 $ yarn deploy
 
-# Starting the app, if the app is not already running 
-# Use port 3300 for production and 3301 for staging 
+# Starting the app, if the app is not already running
+# Use port 3300 for production and 3301 for staging
 $ cd ../../nusmods-export
-$ PORT=3300 NODE_ENV=production yarn start 
+$ PORT=3300 NODE_ENV=production yarn start
```
Comment (ZhangYiJiang) [inline export/README.md]: This needs updating, but we can do it in another PR

File: .vscode/settings.json (hunk 1 above)
Comment (li-kai) [reply .vscode/settings.json]: Nope...I thought that was..god damn it.
